### PR TITLE
run export config with latest patch data

### PR DESCRIPTION
**Pull request by brtietz**
Run export config and api autogen with the latest changes on patch.

### CODE REVIEW

File: api/api_autogen/library/defaults/Battery_FuelCellCommercial.json [hunk 1]
```diff
@@ -169,6 +169,8 @@
 		},
 		"PriceSignal": {
 		},
+		"Revenue": {
```
Comment (mjprilliman) [inline api/api_autogen/library/defaults/Battery_FuelCellCommercial.json:172]: Why is this added to every file?

**Review by mjprilliman — APPROVED** [review]
Just one question for my understanding

File: api/api_autogen/library/defaults/Battery_FuelCellCommercial.json (hunk 1 above)
Comment (brtietz) [reply api/api_autogen/library/defaults/Battery_FuelCellCommercial.json:172]: That comes from https://github.com/NREL/ssc/pull/787/files - which is an ssc vartable input group for revenue forecasting.
